corpus/accounts/urls.py

```diff
@@ -30,4 +32,6 @@
         PasswordResetCompleteView.as_view(),
         name="password_reset_complete",
     ),
+    path("profile/<roll_no>", profile, name="accounts_profile"),
+    path("profile/edit/<roll_no>", edit_profile, name="edit_profile")
```

Comment:
Follow `/resource/id/action` styles for URLs. So, the edit URL would be `/profile/201EC106/edit`